react-native-aztec/src/AztecView.js

```diff
@@ -34,6 +34,14 @@ class AztecView extends React.Component {
     ...TextViewPropTypes, // include the default view properties
   }
 
+  componentDidMount() {
+    TextInputState.registerInput(ReactNative.findNodeHandle(this))
```

Comment:
Could you explain a bit how this registerInput/unregisterInput works and how do they contribute to the solution?

Reply:
TBH I don't think that they are needed in that solution but I added it because I realized that we don't to that at this moment. They add or remove `fieldID` to/from `Set` of inputs in `TextInputState` 
https://github.com/facebook/react-native/blob/master/Libraries/Components/TextInput/TextInputState.js#L85-L95

W/o this the `false` will be returned when `isTextInput` is called.
https://github.com/facebook/react-native/blob/master/Libraries/Components/TextInput/TextInputState.js#L93-L95

Reply:
This sounds really nice but I am a bit afraid about consequences. We don't know who is calling `isTextInput` and what could be done differently if it turns true this time. So I'd avoid adding it unless we have enough context. I am just trying to be extra cautious since this has been a pretty fragile component so far.

Reply:
Ok, I understand your worries so I will remove it then :)